server/bleep/src/webserver/answer/llm_gateway.rs

```diff
@@ -143,16 +143,18 @@ impl Client {
         for _ in 0..self.max_retries {
             match self.chat_oneshot(messages).await {
                 Err(ChatError::TooManyRequests) => {
-                    warn!("LLM request failed, retrying ...");
+                    warn!("too many LLM requests, retrying in {delay:?}...");
                     tokio::time::sleep(delay).await;
                     delay = Duration::from_millis((delay.as_millis() as f32 * SCALE_FACTOR) as u64);
                 }
                 Err(ChatError::BadRequest) => {
-                    warn!("LLM request failed, request not eligible for retry");
+                    debug!("LLM message list: {messages:?}");
```

Comment:
It would be more idiomatic to use `error!(?messages, "LLM request failed")` instead of an additional `debug!`.

Reply:
Wouldn't this cause the message list to log as a debug string to the console every time this error occurs? This means we will get large error strings even when `BLOOP_LOG=bleep=debug` is *not* set, or similar.

Reply:
This is true, it would be helpful to add a comment about this, as it's a bit unintuitive.